ibtk/include/ibtk/StandardTagAndInitStrategySet.h

```diff
@@ -340,7 +340,7 @@ class StandardTagAndInitStrategySet : public SAMRAI::mesh::StandardTagAndInitStr
     /*!
      * \brief The set of SAMRAI::xfer:StandardTagAndInitStrategy objects.
      */
-    std::vector<SAMRAI::mesh::StandardTagAndInitStrategy<NDIM>*> d_strategy_set;
+    std::vector<std::unique_ptr<SAMRAI::mesh::StandardTagAndInitStrategy<NDIM> > > d_strategy_set;
```

Comment:
We should now be able to get rid of `d_managed` since we are managing memory now.

Reply:
This will require changing the interface for the constructor -- currently callers get to decide if they want to hand over management.